flags/tests/test_middleware.py

```diff
@@ -0,0 +1,22 @@
+from django.test import RequestFactory, TestCase, override_settings
+
+from flags.middleware import FlagConditionsMiddleware
+
+
+@override_settings(
+    MIDDLEWARE=(
+        'flags.middleware.FlagConditionsMiddleware',
+    )
+)
+class FlagConditionsMiddlewareTestCase(TestCase):
```

Comment:
Can this inherit from [`SimpleTestCase`](https://docs.djangoproject.com/en/2.1/topics/testing/tools/#simpletestcase) as it doesn't need to access the database?